File: NH/Java/Exercicio_005/Exercicio_005.java
import java.util.Scanner;

class Exercicio_005{
	
	public static void main(String[] args){
		Scanner sc = new Scanner(System.in);
		
		System.out.println("Nota 1: ");
		double nota1 = sc.nextDouble();
		
		System.out.println("Nota 2: ");
		double nota2 = sc.nextDouble();
		
		double media = (nota1 + nota2)/2;
		
		System.out.println("A media entre " +nota1 +" e " +nota2 +" e igual a " +media);
		
		sc.close();
	}
	
}

File: FH/Python/Exercicio_005.py
from decimal import Decimal

nota_1 = Decimal(input("Nota 1: "))
nota_2 = Decimal(input("Nota 2: "))

print(f"A média entre {nota_1} e {nota_2} é igual a {((nota_1 + nota_2) / 2):2f}.")